src/app/page.js

```diff
@@ -0,0 +1,24 @@
+"use client";
+import styles from "@/app/page.module.css";
+
+import TeamCard from "@/team-card/TeamCard";
+
+let test_team = [
+  { name: "Michael Jia", role: "Product Manager", profilePicture: "/cat.jpeg" },
+  { name: "Michael Jia", role: "Product Manager", profilePicture: "/cat.jpeg" },
+  { name: "Michael Jia", role: "Product Manager", profilePicture: "/cat.jpeg" },
+  { name: "Michael Jia", role: "Product Manager", profilePicture: "/cat.jpeg" },
+  { name: "Michael Jia", role: "Product Manager", profilePicture: "/cat.jpeg" },
+  { name: "Michael Jia", role: "Product Manager", profilePicture: "/cat.jpeg" },
+  { name: "Michael Jia", role: "Product Manager", profilePicture: "/cat.jpeg" },
+];
+
+export default function Home() {
+  return (
+    <main className={styles.main}>
```

Comment:
I realized I was getting overlaps because of the styles here. Remove this line + the import and it works perfectly :3

Reply:
Will do, forgor to remove it sorry about that 😅